dallinger/recruiters.py

```diff
@@ -681,22 +680,37 @@ def reward_bonus(self, assignment_id, amount, reason):
 
     @property
     def is_in_progress(self):
-        # Has this recruiter resulted in any participants?
-        return bool(Participant.query.filter_by(recruiter_id=self.nickname).first())
+        """Does an MTurk HIT for the current experiment ID already exist?"""
+        experiment_id = self.config.get("id")
+        hits = self.mturkservice.get_hits(
+            hit_filter=lambda h: h["annotation"] == experiment_id
+        )
+        try:
+            next(hits)
+        except StopIteration:
+            return False
+        else:
+            return True
```

Comment:
This seems a little awkward to me, why not:

```suggestion
        for _ in hits:
            return True
        return False
```